lib/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -95,8 +109,10 @@ Ufe::SceneItem::Ptr ProxyShapeHierarchy::sceneItem() const
 bool ProxyShapeHierarchy::hasChildren() const
 {
 	const UsdPrim& rootPrim = getUsdRootPrim();
-	if (!rootPrim.IsValid())
+	if (!rootPrim.IsValid()) {
+        UFE_LOG("invalid root prim in ProxyShapeHierarchy::hasChildren()");
 		return false;
+    }
```

Comment:
Looks like maybe a space missing before the brace?

Reply:
Tabs vs spaced, tabified in
https://github.com/Autodesk/maya-usd/pull/237/commits/f53f29c7258bc1d99897d843dbfef2e0316dd702